src/webhook/mutation/pod_mutator.go

```diff
@@ -224,6 +224,10 @@ func (m *podMutator) Handle(ctx context.Context, req admission.Request) admissio
 		return *dkResponse
 	}
 
+	if dk.FeatureDisableMetadataEnrichment() {
+		injectionInfo.features[DataIngest] = false
```

Comment:
can this be moved to a function that enables/disables all the features for the `injectionInfo`?

Reply:
injectionInfo is created before getting the dynakube, so to move this logic up we would always need to get the namespace and dynakube to create the `injectionInfo` which is not nice